src/assets/images/sidebar/SidebarIcons.tsx

```diff
@@ -72,6 +72,20 @@ export const SidebarIcons: React.FC<SidebarIconsProps> = (props: SidebarIconsPro
                     <path d="M42.6682 19.4763L32.3318 55.5236" stroke="var(--icons)" strokeWidth="3" strokeLinecap="round" strokeLinejoin="round"/>
                 </svg>
             );
+        case 'internal_transfer':
+            return (
+                <svg width="24" height="24" className={props.className} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <g clip-path="url(#clip0)">
+                    <path d="M9.92725 17.033V20.7047C9.92725 20.9612 10.0919 21.1884 10.3357 21.2692C10.3967 21.289 10.4592 21.2985 10.521 21.2985C10.7062 21.2985 10.8852 21.2114 10.9992 21.0562L13.1469 18.1334L9.92725 17.033Z" fill="#737F92"/>
+                    <path d="M21.7507 3.20031C21.5686 3.07126 21.3295 3.05385 21.1316 3.15756L3.31908 12.4596C3.10849 12.5697 2.9842 12.7945 3.00162 13.0312C3.01983 13.2687 3.17737 13.4714 3.40141 13.5482L8.35329 15.2408L18.8991 6.22368L10.7386 16.0554L19.0376 18.8919C19.0994 18.9125 19.1643 18.9236 19.2292 18.9236C19.3369 18.9236 19.4437 18.8943 19.538 18.8373C19.6884 18.7455 19.7905 18.5911 19.8166 18.4177L21.9937 3.77189C22.0262 3.55022 21.9327 3.33014 21.7507 3.20031Z" fill="#737F92"/>
+                    </g>
+                    <defs>
+                    <clipPath id="clip0">
+                    <rect width="19" height="19" fill="white" transform="translate(3 3.09023)"/>
+                    </clipPath>
```

Comment:
Can you fix indentation?
Also maybe you need to use color env for fill attribute?